koku/masu/test/processor/aws/test_aws_report_processor.py

```diff
@@ -944,7 +940,7 @@ def test_delete_line_items_success(self):
         for bill_id in bill_ids:
             with schema_context(self.schema):
                 before_count = self.accessor.get_lineitem_query_for_billid(bill_id).count()
-            result = processor._delete_line_items(AWSReportDBAccessor, self.column_map)
+            result = processor._delete_line_items(AWSReportDBAccessor, REPORT_COLUMN_MAP)
```

Comment:
Do we need to pass in `REPORT_COLUMN_MAP` through the constructor to the `_delete_line_items` method? I think that we import the `REPORT_COLUMN_MAP` in the `ReportProcessorBase` class and use it directly in at least one other method of that class so we should also do this with `_delete_line_items` to be consistent.

Reply:
I agree. So in general, REPORT_COLUMN_MAP should not be passed into a method or constructor.  It's a constant that doesn't change.  Parameters are for data that changes.

Reply:
updated @kholdaway @abaiken